spec.bs

```diff
@@ -1563,3 +1598,133 @@ specification adds another value for fenced frames:
+
+  4. If |container| is a [=fenced navigable container=], then for each |feature| supported,
+
+      1. If |feature| exists in |required features|, set |inherited policy|[|feature|] to "`Enabled`".
+
+        Otherwise, set |inherited policy|[|feature|] to "`Disabled`".
+
+    Otherwise, for each |feature| supported,
+
+      1. Let |isInherited| be the result of running [=Define an inherited policy for feature in
+        container at origin=] on |feature|, |container| and <var ignore>origin</var>.
+
+      2. Set |inherited policy|[|feature|] to |isInherited|.
+</div>
+
+<div algorithm=allow-attribute-fenced-frame>
+  Modify the [=The allow attribute of the iframe element=] section.
+
+  Rename it to "The allow attribute of the iframe and fencedframe element".
+
+  Rewrite the section to read:
+
+  &lt;iframe> and &lt;fencedframe> elements have an "allow" attribute, which contains an
+  [=ASCII-serialized policy directive=].
+
+  The allowlist for the features named in the attribute may be empty; in that case, the default
+  value for the allowlist is "`src`", which represents the origin of the URL in the iframe’s src
+  attribute, or the fencedframe's [=fenced frame config=].
+
+  When not empty, the "allow" attribute will result in adding an allowlist for each recognized
+  feature to the iframe element’s content navigable's container policy or the fencedframe element'same
+  [=fenced navigable container/fenced navigable=]'s container policy, when it is constructed.
+</div>
+
+<div algorithm=create-permissions-policy-response>
+  Modify the definition of [=Create a Permissions Policy for a navigable from response=] to read:
+
+  Given null, a [=navigable container=], or a [=fenced navigable container=] (|container|), an
+  [=origin=] (|origin|), a [=response=] (<var ignore>response</var>), and null or a
+  list of [=features=] (|required features|), this algorithm returns a new
+  [=Document/permissions policy=].
+
+  Modify step 1 of the algorithm to read:
+  1. Let <var ignore>policy</var> be the result of running
+    [=Create a Permissions Policy for a navigable=] given |container|, |origin|, and
+    |required features|.
+</div>
+
+<div algorithm=shared-document-creation-changes>
+  Modify the [=shared document creation infrastructure=] algorithm. Rewrite step 3 to read:
+
+  3. Let <var ignore>permissionsPolicy</var> be the result of [=Create a Permissions Policy for a
+    navigable from response|creating a permissions policy from a response=] given
+    |navigationParams|'s [=navigable=]'s [=navigable container|container=], |navigationParams|'s
+    [=navigation params/origin=], |navigationParams|'s [=navigation params/response=], and
+    <span class=XXX>TODO: Populate this with the required permissions to load from the config in
+    the navigation params</span>.
+</div>
+
+<div algorithm=new-browsing-context-changes>
+  Modify the [=create a new browsing context and document=] algorithm. Rewrite step 7 to read:
```

Comment:
Let's briefly mention that these changes are made _in addition to_ the changes made in https://wicg.github.io/fenced-frame/#creating-browsing-contexts-patch (please use https://speced.github.io/bikeshed/#section-links to link to that section).